src/test/java/com/epam/ta/reportportal/dao/suite/WidgetRepositoryTest.java

```diff
@@ -23,13 +23,15 @@
 import com.epam.ta.reportportal.commons.querygen.ProjectFilter;
 import com.epam.ta.reportportal.dao.WidgetRepository;
 import com.epam.ta.reportportal.entity.widget.Widget;
+import org.junit.Assert;
```

Comment:
I suppose it should be replaced with junit5

Reply:
done